integration-cli/docker_cli_build_unix_test.go

```diff
@@ -28,8 +28,13 @@ func (s *DockerSuite) TestBuildResourceConstraintsAreUsed(c *check.C) {
 
 	cID := strings.TrimSpace(out)
 
+	cfg, err := inspectFieldJSON(cID, "HostConfig")
+	if err != nil {
+		c.Fatal(err)
+	}
+
 	type hostConfig struct {
-		Memory     int64
+		Memory     float64
```

Comment:
This could be another example of why we may have to bite the bullet and covert the externalized types. The fact that this shows up as a float instead of an int is really an artifact of Go's JSON deserializing code and I'm not convinced we should expect users to know or care about that.